packages/web/src/scss/components/SegmentedControl/index.html

```diff
@@ -0,0 +1,329 @@
+
+<section class="py-900 py-tablet-1000">
+	<div class="Container">
+		<h2 class="docs-Heading">Multiple Selection</h2>
+		<div class="docs-Stack docs-Stack--start">
+
+			{{#each @root.variants as |variant|}}
+
+			<fieldset class="SegmentedControl SegmentedControl--{{variant}}">
+
+				<legend class="accessibility-hidden">Label</legend>
+
+				{{#each @root.items as |item|}}
+
+				<input type="checkbox"
+					id="segmentedControl-multiple-{{variant}}-{{item}}"
+					name="segmented-{{variant}}"
+					value="value-{{item}}"
+					class="SegmentedControlItem__input"
+					{{#if (eq item "1")}}checked{{/if}}
+				>
+				<label for="segmentedControl-multiple-{{variant}}-{{item}}"
+					class="SegmentedControlItem__label"
+				>
+					<span class="text-truncate">Label</span>
+				</label>
+
+				{{/each}}
+
+			</fieldset>
+
+			{{/each}}
+
+		</div>
+	</div>
+</section>
+
+<section class="py-900 py-tablet-1000">
+	<div class="Container">
+		<h2 class="docs-Heading">Variant with Disabled Item</h2>
+		<div class="docs-Stack docs-Stack--start">
+
+			{{#each @root.variants as |variant|}}
+
+			<fieldset class="SegmentedControl SegmentedControl--{{variant}}">
+
+				<legend class="accessibility-hidden">Label</legend>
+
+				{{#each @root.items as |item|}}
+
+				<input type="radio"
+					id="segmentedControl-disabled-{{variant}}-{{item}}"
+					name="segmented-{{variant}}"
+					value="value-{{item}}"
+					class="SegmentedControlItem__input"
+					{{#if (eq item "1")}}checked{{/if}}
+				  {{#if (eq item "2")}}disabled{{/if}}
+				>
+				<label for="segmentedControl-disabled-{{variant}}-{{item}}"
+					class="SegmentedControlItem__label"
+				>
+					<span class="text-truncate">Label</span>
+				</label>
+
+				{{/each}}
+
+			</fieldset>
+
+			{{/each}}
+
+		</div>
+	</div>
+</section>
+
+<section class="py-900 py-tablet-1000">
+	<div class="Container">
+		<h2 class="docs-Heading">Fluid</h2>
+		<div class="docs-Stack docs-Stack--start">
+
+			<fieldset class="SegmentedControl SegmentedControl--fill SegmentedControl--fluid">
+				<legend class="accessibility-hidden">Label</legend>
+
+				{{setVar "items" "1" "2" "3" "4" "5"}}
+
+				{{#each @root.items as |item|}}
+
+				<input type="radio"
+					id="segmented-control-label-stretch-{{item}}"
+					name="segmented-stretch-filled"
+					value="value-{{item}}"
+					class="SegmentedControlItem__input"
+					{{#if (eq item "1")}}checked{{/if}}
+				>
+				<label for="segmented-control-label-stretch-{{item}}"
+					class="SegmentedControlItem__label"
+				>
+					<svg width="20" height="20" aria-hidden="true">
+						<use xlink:href="/assets/icons/svg/sprite.svg#file"></use>
+					</svg>
+					<span class="text-truncate">Label {{item }} {{#if (eq item "2")}} - this label will be truncated{{/if}}</span>
+				</label>
+
+				{{/each}}
+
+			</fieldset>
+
+		</div>
+	</div>
+</section>
+
+<section class="py-900 py-tablet-1000">
+	<div class="Container">
+		<h2 class="docs-Heading">Variant with Tooltip</h2>
```

Comment:
You made this work even with keyboard control! 👏🏻 

<img width="362" alt="obrazek" src="https://github.com/user-attachments/assets/30b156cb-20fa-40a9-8dce-ae44933ee103" />